packages/main/src/components/AnalyticalTable/util/index.ts

```diff
@@ -22,3 +24,41 @@ export const orderByFn = (rows, functions, directions) => {
 
   return defaultOrderByFn(rows, wrappedSortfunctions, directions);
 };
+
+export const toggleSelected = (e, row, webComponentsReactProperties, dispatch, toggleRowSelected, selectedFlatRows) => {
```

Comment:
The function itself is fine - did you also think of creating it on the row hook itself? This would help you to get rid of all parameters except `e`